Hw4_T1/Hw4_T1/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hw4_T1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            for (int i = (char)90 ; i >= (char)65; i--)
```

Comment:
90, 65 - these are magic numbers. Do you know why? Is NO - you can ask me - I will explain you